include/s3select_oper.h

```diff
@@ -1316,7 +1324,7 @@ class variable : public base_statement
 
       memcpy(&m_star_op_result_charc[pos], m_scratch->get_column_value(i).data(), len);
       pos += len;
-      m_star_op_result_charc[ pos ] = ',';//TODO need for another abstraction (per file type)
+      m_star_op_result_charc[ pos ] = column_delimiter;
```

Comment:
the current code state is not enabling a star-operation (as a generic operation for all different input types, CSV,Parquet,JSON and other) 
hence the comment,  there is a need for some abstraction.

the current state enables start-operation for CSV (for the parquet branch it needs for some modification)